docs/index.md

```diff
@@ -4,14 +4,23 @@ description: EthSigner is an open-source private transaction manager developed u
 
 # EthSigner Transaction Signer
 
-EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. 
+## What is Ethsigner?
 
-EthSigner provides private key signing to separate key management from transaction validation. EthSigner supports [Hashicorp Vault](Using-EthSigner/Hashicorp.md) 
-and [Azure Key Vault](Using-EthSigner/Azure.md). 
+EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. It adds a layer of security by signing transactions using a private key that can be secured in a variety of cloud providers, or encrypted on a local disk. EthSigner therefore separates private key management from transaction validation. 
```

Comment:
Editing comments - I'd rewrite the second sentence to remove 'It' (it's not always clear what it/this refer to without rereading the preceding sentence): 
A layer of security is added by signing transactions using a private key secured in a cloud provider, or encrypted on a local disk.

I think 'therefore' could be removed from the last sentence

Reply:
I'll combine the last 2 sentences so that it reads, "EthSigner separates private key management from transaction validation by signing transactions using a private key that can be secured in a variety of cloud providers, or encrypted on a local disk."